src/Infrastructure.EntityFramework/Repositories/DatabaseContext.cs

```diff
@@ -28,6 +28,7 @@ public DatabaseContext(DbContextOptions<DatabaseContext> options)
     public DbSet<UserServiceAccountAccessPolicy> UserServiceAccountAccessPolicy { get; set; }
     public DbSet<GroupServiceAccountAccessPolicy> GroupServiceAccountAccessPolicy { get; set; }
     public DbSet<ApiKey> ApiKeys { get; set; }
+    public DbSet<Cache> Cache { get; set; }
```

Comment:
Most of these use the plural name of the item for the property (other than the SM ones) but I think `Cache` _is_ the plural, or at least `Caches` sound weird to me.

Reply:
Cache seems appropriate to me still